### PR TITLE
Adding Uighur Locale 

### PR DESCRIPTION
**Pull request by abduwaly**

### CODE REVIEW

File: src/locale/ui/_lib/formatDistance/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    result = formatDistanceLocale[token].one
+  } else {
+    result = formatDistanceLocale[token].other.replace('{{count}}', count)
+  }
+
+  if (options.addSuffix) {
+    if (options.comparison > 0) {
+      return result
+    } else {
+      return result + 'بولدى'
```
Comment (abduwaly) [inline src/locale/ui/_lib/formatDistance/index.js]:
```suggestion
      return result + 'بولدى '
```

File: src/locale/ui/_lib/formatDistance/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    result = formatDistanceLocale[token].one
+  } else {
+    result = formatDistanceLocale[token].other.replace('{{count}}', count)
+  }
+
+  if (options.addSuffix) {
+    if (options.comparison > 0) {
+      return result
+    } else {
+      return result + 'بولدى '
```
Comment (abduwaly) [inline src/locale/ui/_lib/formatDistance/index.js]:
```suggestion
      return result + ' بولدى'
```

**Comment by abduwaly** [comment]
Anyone help approve ? @kossnocorp @leshakoss

**Comment by kossnocorp** [comment]
I've released the change as `v2.0.0-alpha.29`: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha29

**Pull request merged**